docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  mysql:
+    image: mysql:latest
+    container_name: mysql_container
+    environment:
+      MYSQL_ROOT_PASSWORD: 1234
+      MYSQL_DATABASE: app_db
+      MYSQL_USER: app_user
+      MYSQL_PASSWORD: 1234
```

Comment:
Similarly, using '1234' as `MYSQL_PASSWORD` is insecure. It's advisable to use environment variables or Docker secrets for passwords.